Library/ViewModels/DashboardViewModel.swift

```diff
@@ -32,6 +32,12 @@ public protocol DashboardViewModelInputs {
   /// Call when the projects drawer has animated out.
   func dashboardProjectsDrawerDidAnimateOut()
 
+  /// Call to open message thread for specific project
+  func messageThreadNavigated(projectId: Param, messageThread: MessageThread)
+
+  /// Call to open project messages thread
+  func openMessageThreadRequested()
```

Comment:
Our commands are linked to direct inputs, so unless we're reusing this command (which is the exception more than the rule for us, I think), let's be as descriptive as possible per the actual action/input.